lib/Notifications/Notifier.php

```diff
@@ -78,6 +78,16 @@ public function prepare(INotification $notification, string $languageCode): INot
 					$l->t('A new login into your account was detected. The IP address %s was classified as suspicious. If this was you, you can ignore this message. Otherwise you should change your password.', $suspiciousIp)
 				);
 
+				// Add button for more information about the IP-address
+				$action = $notification->createAction();
+				$label = $l->t('More information');
+				$link = 'https://iplookup.flagfox.net/?ip=' . $suspiciousIp;
```

Comment:
cc @karlitschek

Reply:
@nickvergessen I've modified the description of the button now. Hope that makes more clear what it does.
See a46eb50

Reply:
Interesting. I like this feature from a user perspective. Of course we have to be careful regarding the sharing of data. But technically we wouldn't share the IP of the user but the IP of the suspicious login attempt which is probably another user, right? Otherwise it probably o.k. to have a text below the button which explains that this links to an external service. What do you think?

Reply:
I'd say in theory it only reports the IP of another user but in reality it doesn't work in 100% of the cases so it is definitely possible that it reports your own IP as suspicious. So we probably need to make clear that you will share the ip address with this 3rdparty service. 

As far as I know isn't it possible to put something below the button so maybe we could change the message where we inform the user that it will share the ip address with the 3rdparty if you click on the button?

What do you think?

Reply:
Here an attempt which shall make clear what the button does:
https://github.com/nextcloud/suspicious_login/pull/535/commits/c4fd2e67de3c1934784cbb86252a67b103443e5b

Reply:
I still believe that the integration of this external site should be options. Some admins might be fine with their users sending potential sensitive information to a third party, some won't.

I would rather let the admin decide if the feature is active than the user. If a user doesn't know how to look up the geo location of an IP address then they also won't understand the implications of sending the IP to an external website.

Reply:
I've made it now a configurable option. Please have a look again :)